ledger/sandbox/src/main/scala/com/digitalasset/platform/sandbox/SandboxServer.scala

```diff
@@ -162,10 +162,14 @@ class SandboxServer(actorSystemName: String, config: => SandboxConfig) extends A
       (mbLedgerTime, config.timeProviderType) match {
         case (None, TimeProviderType.WallClock) => (TimeProvider.UTC, None)
         case (None, _) =>
-          val ts = TimeServiceBackend.simple(Instant.EPOCH)
+          val ts = TimeServiceBackend.simple(
+            Instant.EPOCH,
+            config.timeProviderType == TimeProviderType.StaticAllowBackwards)
           (ts, Some(ts))
         case (Some(ledgerTime), _) =>
-          val ts = TimeServiceBackend.simple(ledgerTime)
+          val ts = TimeServiceBackend.simple(
+            ledgerTime,
```

Comment:
You can combine the last 2 branches and simply do `mbLedgerTime.getOrElse(Instant.EPOCH)`.

Reply:
right, will do that.